### DISCUSSION TITLE
GitHub Copilot and GitHub Copilot Chat stopped working

### DISCUSSION THREAD

**Discussion by bholemaharaj** [community/community#134876] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

At my work we have licenses for GitHub Copilot and chat. When we started using Copilot, we authenticated once using Okta and then installed the GitHub Copilot VS Code extension along with chat. Signed in to GitHub Copilot and everything was working just fine.

Last week Thursday, Copilot and chat stopped working and it will not sign me in. 
VS Code shows as if I am logged in but the chat windows keep asking to sign in. Here is my diagnostic log and out put showing in the terminal.

Any help would be appreciated.

Output in terminal:

```
2024-08-06 12:13:57.929 [info] [auth] GitHub login failed
2024-08-06 12:13:57.930 [info] [chat] copilot token chat_enabled: undefined
2024-08-06 12:13:57.930 [info] [RemoteAgents] Unable to fetch remote agents because user is not signed in.
2024-08-06 12:13:57.932 [info] [auth] GitHub login failed
2024-08-06 12:13:57.936 [info] [auth] GitHub login failed
2024-08-06 12:13:57.937 [info] [ContextKeysContribution] You are not signed in to GitHub. Please sign in to use Copilot.
2024-08-06 12:13:57.937 [info] [auth] GitHub login failed
2024-08-06 12:13:57.938 [info] [LanguageModelAccess] LanguageModel/Embeddings are not available without auth token
2024-08-06 12:13:57.938 [info] [LanguageModelAccess] Error: GitHubLoginFailed
    at RS.authShowWarnings (c:\Users\<userName>\.vscode\extensions\github.copilot-chat-0.18.1\dist\extension.js:633:13828)
    at RS.getCopilotToken (c:\Users\<userName>\.vscode\extensions\github.copilot-chat-0.18.1\dist\extension.js:633:11894)
    at Q_.getCopilotToken (c:\Users\<userName>\.vscode\extensions\github.copilot-chat-0.18.1\dist\extension.js:144:2645)
    at PS._getAuthSession (c:\Users\<userName>\.vscode\extensions\github.copilot-chat-0.18.1\dist\extension.js:634:2188)
    at r (c:\Users\<userName>\.vscode\extensions\github.copilot-chat-0.18.1\dist\extension.js:634:627)
```

Diagnostic Log:

```

## Copilot

- Version: 1.220.0
- Build: prod
- Editor: vscode/1.92.0

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Send Restricted Telemetry: disabled
- Chat: enabled

## Node setup

- Number of root certificates: 147
- Operating system: Windows_NT
- Operating system version: 10.0.22631
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: HelixFetcher

## Reachability

- github.com: HTTP 200
- api.github.com: HTTP 200
- copilot-proxy.githubusercontent.com: HTTP 200
- api.githubcopilot.com: HTTP 200
- default.exp-tas.com: HTTP 200

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy authentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

```
![image (1)](https://github.com/user-attachments/assets/c6b2d477-1892-444b-b35e-560c478f0d3a)
![image](https://github.com/user-attachments/assets/cd9f379c-7599-42ac-a9be-f670e26b55c0)

**Comment by lucasgabrielcastro** [comment]
Same issue, but I do have a personal license for Copilot, not from an org.

I am facing this same issue: Github does not authenticate for copilot only

this is the output for Copilot Chat extension:
![Settings_—_Blog](https://github.com/user-attachments/assets/1a9e541b-b411-40d2-8b73-3f8f1cbd8fea)

this is the output for just the Copilot extension:
![Settings_—_Blog-3](https://github.com/user-attachments/assets/cd5ec41e-aecd-4e0c-a1a2-8bd6939518f8)


System info:

macOS (Sonoma 14.6)
M1 Pro
16GB RAM
Visual Studio Code, on its latest version:
Version: 1.92.0 (Universal)
Commit: b1c0a14de1414fcdaa400695b4db1c0799bc3124
Date: 2024-07-31T23:26:45.634Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Darwin arm64 23.6.0

any assistance is greatly appreciated.

Thank you in advance.

**Comment by lucasgabrielcastro** [comment, marked as answer]
@bholemaharaj on my VSCode Settings and deleted my preferred GitHub account, and it just worked. 

![Settings_—_Blog-4](https://github.com/user-attachments/assets/8247e2c6-21b3-4f32-a978-9c7b47e6c659)

Steps:

on VSCode

- Settings
- type Github
- on the field "Preferred Account"
- erase whatever it is in the field, and close
- redo the steps above and put your username again back in the input field.


Hope this works for you.

**Reply by vdiazh** [reply]
Gracias @lucasgabrielcastro, tenía 3 días de estar con esto, ya totalmente frustrado.

**Reply by HeMuling** [reply]
Can't find this option anymore, is is still available?

**Comment by bholemaharaj** [comment]
Holly cow, thanks @lucasgabrielcastro, that worked like a charm!!! This issue haunted me since Friday and the weekend.

**Reply by lucasgabrielcastro** [reply]
@bholemaharaj you are very welcome!

**Reply by vdiazh** [reply]
Me pasó lo mismo, estaba fustrado.

**Comment by pvgdevelop** [comment]
Thank you @lucasgabrielcastro!

**Comment by txnthanh** [comment]
Thank you @lucasgabrielcastro
This works perfectly. Really appreciate your time sharing with us.
All the best.

**Comment by copdips** [comment]
doesn't work for me yet, hope vscode could enbale more logs for the troubleshooting

**Comment by mir183** [comment]
doesnt work for me :(

**Comment by vhbui02** [comment]
Anyone get this bug? It happens to me this afternoon.
